docs/topics/api/reviews.rst

```diff
@@ -72,8 +72,9 @@ This endpoint allows you to fetch a review by its id.
     :>json int version.id: The add-on version id the review applies to.
     :>json string version.version: The add-on version string the review applies to.
     :>json object user: Object holding information about the user who posted the review.
-    :>json string user.url: The user profile URL.
+    :>json string user.id: The user id.
     :>json string user.name: The user name.
+    :>json string user.url: The user profile URL.
 
```

Comment:
The addons and activity API docs were already mentioning the user id - looks like they were written with the future in mind :)